support/config/types.go

```diff
@@ -0,0 +1,11 @@
+package config
+
+const (
+	// PodSafeToEvictKey is an annotation used by the CA operator which makes sure
```

Comment:
this doesn't seem to be used anywhere, can we drop it?

Reply:
Yeah, makes sense